crates/ibc/src/core/ics02_client/handler/create_client.rs

```diff
@@ -17,15 +17,12 @@ use crate::core::ValidationContext;
 use crate::core::ics02_client::client_state::ClientState;
 use crate::core::ics02_client::client_type::ClientType;
 use crate::core::ics02_client::consensus_state::ConsensusState;
-use crate::core::ics02_client::context::ClientReader;
 use crate::core::ics02_client::error::ClientError;
 use crate::core::ics02_client::events::CreateClient;
-use crate::core::ics02_client::handler::ClientResult;
 use crate::core::ics02_client::height::Height;
 use crate::core::ics02_client::msgs::create_client::MsgCreateClient;
 use crate::core::ics24_host::identifier::ClientId;
 use crate::events::IbcEvent;
-use crate::handler::{HandlerOutput, HandlerResult};
 use crate::timestamp::Timestamp;
 
 /// The result following the successful processing of a `MsgCreateClient` message.
```

Comment:
Should we keep the `CreateClientResult` struct? (not used anymore)

Reply:
[ff3c0f7](https://github.com/cosmos/ibc-rs/pull/456/commits/ff3c0f736586393ea740861f9d411fec619af838)